src/component/project/Project/Project.tsx

```diff
@@ -116,12 +115,10 @@ const Project = () => {
         <div ref={ref}>
             <div className={styles.header}>
                 <div className={styles.innerContainer}>
-                    <h2
-                        data-loading
-                        className={styles.title}
-                        style={{ margin: 0, width: '100%' }}
-                    >
-                        <div className={styles.titleText}>{project?.name}</div>
+                    <h2 className={styles.title}>
+                        <div className={styles.titleText} data-loading>
+                            {project?.name || projectId}
```

Comment:
We should probably not use `projectId` as a fallback for `project.name`, given recent feedback

Reply:
This is so that title takes has a placeholder when loading, that takes space. Otherwise I can add `|| "Project name"`

Reply:
This is so that title takes has a placeholder when loading, that takes space. Otherwise I can add || "Project name"